politeiawww/backend.go

```diff
@@ -492,6 +492,50 @@ func (b *backend) emailResetPasswordVerificationLink(email, token string) error
 	return b.cfg.SMTP.Send(msg)
 }
 
+func (b *backend) emailAllAdminsForNewSubmittedProposal(token string, propName string, username string, userEmail string) error {
+	if b.cfg.SMTP == nil {
+		return nil
+	}
+
+	l, err := url.Parse(b.cfg.WebServerAddress + "/proposals/" + token)
+	if err != nil {
+		return err
+	}
+
+	var buf bytes.Buffer
+	tplData := newProposalSubmittedTemplateData{
+		Link:     l.String(),
+		Name:     propName,
+		Username: username,
+		Email:    userEmail,
+	}
+
+	err = templateNewProposalSubmitted.Execute(&buf, &tplData)
+	if err != nil {
+		return err
+	}
+
+	from := "noreply@decred.org"
+	subject := "New Proposal Submitted"
```

Comment:
I would prefix this with "Politeia -" or something like that.

Reply:
I don't think it is necessary because the email title already contains the "Politeia" reference:
<img width="517" alt="screen shot 2018-10-16 at 3 55 31 pm" src="https://user-images.githubusercontent.com/14864439/47040145-ee987c80-d15b-11e8-8114-94ed9c0fc06b.png">

Reply:
Oh, that's a good point.